src/modules/facility/facility.interface.ts

```diff
@@ -0,0 +1,35 @@
+import { DateOnlyString } from '@ukef/helpers';
+
+export interface Facility {
+  dealIdentifier: string;
+  facilityIdentifier: string;
+  portfolioIdentifier: string;
+  dealBorrowerIdentifier: string;
+  maximumLiability: number;
+  productTypeId: string;
+  capitalConversionFactorCode: string;
+  currency: string;
+  guaranteeCommencementDate: DateOnlyString;
+  guaranteeExpiryDate: DateOnlyString;
+  nextQuarterEndDate: DateOnlyString;
+  facilityInitialStatus: string;
+  facilityOverallStatus: string;
+  delegationType: string;
+  intrestOrFeeRate: number;
```

Comment:
Should we correct the spelling to _interestOrFeeRate_ (and add to the spreadsheet/alert consumers)?

Reply:
This is just in the Mulesoft implementation (it is called `UserDefinedDate1` in ACBS) so I've corrected it.